packages/unmock-cli/README.md

```diff
@@ -9,4 +9,19 @@
 [build]: https://circleci.com/gh/unmock/unmock-js
 [coverage]: https://coveralls.io/github/unmock/unmock-js
 
-This package contains the base functions needed to build or extend the [unmock](../unmock/README.md) CLI.  It also contains the cli itself for testing purposes.
\ No newline at end of file
+This package contains the base functions needed to build or extend the [unmock](../unmock/README.md) CLI.  It also contains the cli itself for testing purposes.
+
+## Usage: unmock `<command>`
+
+### Options:
+  - `-h, --help` output usage information
+
+### Commands:
+  - `init [options] [dirname]` Sets up a new test suite
+    - `--installer [installer]`  Specify Package Manager (yarn/npm)
```

Comment:
In future, maybe the CLI could also check existing `package-lock.json` or `yarn.lock` to automatically determine the manager?

Reply:
I just added a fix: [bd97f36](https://github.com/unmock/unmock-js/pull/327/commits/bd97f36dac08f7f767ca78c22d5d9aa31e8165b2).